### ISSUE TITLE
Can not craft Salis Mundis

### ISSUE THREAD

**Issue by astro013** [TheLimePixel/Modern-Skyblock-3#627]
After updating from 3.3.1.4 to 3.4.1.4, In gated mode I tried to progress through Thaumcraft which was added and am unable to craft salis mundis, I crafted the crystals, broke them with a pickaxe, slept through the night and got the book from Thaumcraft.  I then used three different crystals, a bowl, flint and a piece of redstone in a vanilla crafting table and the recipe did not have an output.

Also the Thaumcraft crystal quest and the trees were automatically marked as completed when I updated the pack.

I tried resetting these quests and re-crafting the crystals, breaking them sleeping through the night but I did not get the book again and could still not craft salis mundis. I assume this is a bug unless I am doing something wrong.

**Comment by ZacEllis** [comment]
Exactly the same as above, I even watched tutorials to make sure I wasn't missing something. Have tried with lots of different combinations of different crystals.

**Comment by TheLimePixel** [comment]
Just sounds like you don't have the stage unlocked.

**Comment by ZacEllis** [comment]
Definitely have the stage unlocked, can do everything else in thaumcraft fine after cheating in some salis mundis

**Comment by TheLimePixel** [comment]
Salis Mundis is the only thing gated in thaumcraft...

**Comment by tloskpd** [comment]
Just tested this myself in a brand new test world created on the 3.4.1.4 version.  I selected kitchen sink mode to unlock on quest chains.  I created all the crystals, broke them, slept and got the book.  The crystal quest was also marked as complete.  Using 3 different crystals, flint, bowl, and piece of redstone did not have an output.

Edit.  I manually reran the /gamestage add @p tw command and was able to craft the Salis Mundis so something is going on with not properly unlocking it.  (but the recipe in JEI shows up as a blank crafting grid with salis mundis as a result)
